pkg/param/render.go

```diff
@@ -151,6 +151,10 @@ func newUndefinedKeyError(err string) error {
 
 // RenderObjectRefs function renders object refs from TemplateParams
 func RenderObjectRefs(in map[string]crv1alpha1.ObjectReference, tp TemplateParams) (map[string]crv1alpha1.ObjectReference, error) {
+	if tp.Time == "" {
```

Comment:
Was this missing? Is it something we need?

Reply:
in the cases where we don't want to pass an actual template param (for example in validate blueprint case), this function fails.
Using this we are checking that `tp` is empty or not, we we create template param from our code we do specify time.